pkg/cluster/internal/providers/docker/network.go

```diff
@@ -50,8 +50,11 @@ func ensureNetwork(name string, ipFamily config.ClusterIPFamily) error {
 	if string(out) == name+"\n" {
 		return nil
 	}
+	// TODO: ipv6 subnet should probably not be fixed
+	// Though maybe just require the user to handle this by creating the network
+	// as they desire before running kind ...
 	if ipFamily == config.IPv6Family {
-		return exec.Command("docker", "network", "create", "-d=bridge", "--ipv6", name).Run()
+		return exec.Command("docker", "network", "create", "-d=bridge", "--ipv6", "--subnet=fc00:db8:1::/64", name).Run()
```

Comment:
we can use a /104 , it means 255 addresses and we have a low risk of collisions based on this https://github.com/kubernetes/kubernetes/pull/90115#issuecomment-616670315

Reply:
i'm using a /104 just above this now. i'm not sure this is the value we want to settle on.

Reply:
test are running though https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kind/1508/pull-kind-conformance-parallel-1-16/1253795238921637894